Not working on SDK 30

Caused by: java.lang.NoClassDefFoundError: Failed resolution of: Ljunit/framework/Assert;
        at abak.tr.com.boxedverticalseekbar.BoxedVertical.init(BoxedVertical.java:136)
        at abak.tr.com.boxedverticalseekbar.BoxedVertical.<init>(BoxedVertical.java:105)
        	... 38 more
     Caused by: java.lang.ClassNotFoundException: Didn't find class "junit.framework.Assert" on path: DexPathList[[zip file "/data/app/~~95EQU6LLgq3EYOt3yBXX1Q==/com.silkyapps.maclock-dU2DoBl7bjofNJ7RM5VJxQ==/base.apk"],nativeLibraryDirectories=[/data/app/~~95EQU6LLgq3EYOt3yBXX1Q==/com.silkyapps.maclock-dU2DoBl7bjofNJ7RM5VJxQ==/lib/arm64, /data/app/~~95EQU6LLgq3EYOt3yBXX1Q==/com.silkyapps.maclock-dU2DoBl7bjofNJ7RM5VJxQ==/base.apk!/lib/arm64-v8a, /system/lib64, /system_ext/lib64]]
        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:207)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
        	... 40 more

--- a/BoxedVerticalSeekBarLib/src/main/java/abak/tr/com/boxedverticalseekbar/BoxedVertical.java
+++ b/BoxedVerticalSeekBarLib/src/main/java/abak/tr/com/boxedverticalseekbar/BoxedVertical.java
@@ -24,7 +24,6 @@ import android.view.Display;
 import android.view.MotionEvent;
 import android.view.View;
 
-import junit.framework.Assert;
 
 public class BoxedVertical extends View{
     private static final String TAG = BoxedVertical.class.getSimpleName();
@@ -133,9 +132,9 @@ public class BoxedVertical extends View{
             mImageEnabled = a.getBoolean(R.styleable.BoxedVertical_imageEnabled, mImageEnabled);
 
             if (mImageEnabled){
-                Assert.assertNotNull("When images are enabled, defaultImage can not be null. Please assign a drawable in the layout XML file", a.getDrawable(R.styleable.BoxedVertical_defaultImage));
-                Assert.assertNotNull("When images are enabled, minImage can not be null. Please assign a drawable in the layout XML file", a.getDrawable(R.styleable.BoxedVertical_minImage));
-                Assert.assertNotNull("When images are enabled, maxImage can not be null. Please assign a drawable in the layout XML file", a.getDrawable(R.styleable.BoxedVertical_maxImage));
+                assert a.getDrawable(R.styleable.BoxedVertical_defaultImage) != null : "When images are enabled, defaultImage can not be null. Please assign a drawable in the layout XML file";
+                assert a.getDrawable(R.styleable.BoxedVertical_minImage) != null : "When images are enabled, minImage can not be null. Please assign a drawable in the layout XML file";
+                assert a.getDrawable(R.styleable.BoxedVertical_maxImage) != null : "When images are enabled, maxImage can not be null. Please assign a drawable in the layout XML file";
 
                 mDefaultImage = ((BitmapDrawable) a.getDrawable(R.styleable.BoxedVertical_defaultImage)).getBitmap();
                 mMinImage = ((BitmapDrawable) a.getDrawable(R.styleable.BoxedVertical_minImage)).getBitmap();

